server/src/main/java/org/apache/iotdb/db/qp/physical/sys/LoadDataPlan.java

```diff
@@ -48,6 +48,15 @@ public LoadDataPlan(String inputFilePath, String measureType) {
     return ret;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    List<String> ret = new ArrayList<>();
+    if (measureType != null) {
+      ret.add(measureType);
+    }
+    return ret;
```

Comment:
Here too.

Reply:
I'll change it. Thanks for your suggestion.